hack/ci/Vagrantfile-flatcar

```diff
@@ -0,0 +1,118 @@
+      "--cpus", cpus 
+    ]
+    v.check_guest_additions = false
+    v.functional_vboxsf = false
+  end
+
+  config.vm.provider :libvirt do |v|
+    v.memory = memory
+    v.cpus = cpus
+  end
+
+  config.ssh.username = 'core'
+  config.ssh.insert_key = true
+  config.ssh.forward_agent = true
+
+  config.vm.synced_folder ".", "/vagrant", type: "rsync"
+
+  config.vm.provision "install-dependencies", type: "shell", run: "once" do |sh|
+    sh.inline = <<~SHELL
+      set -euxo pipefail
+
+      # Install Go
+      GO_VERSION=1.18
+      mkdir /opt/go
+      curl -sSfL -o- https://golang.org/dl/go$GO_VERSION.linux-amd64.tar.gz | tar xfz - -C /opt
+
+      # Setup Networking 
+      systemctl enable docker
+      modprobe br_netfilter
+
+      cat <<EOF | tee /etc/modules-load.d/k8s.conf
+      br_netfilter
+      EOF
+
+      cat <<EOF | tee /etc/sysctl.d/k8s.conf
+      net.bridge.bridge-nf-call-ip6tables = 1
+      net.bridge.bridge-nf-call-iptables = 1
+      EOF
+      sysctl --system
+
+      # Install k8s components
+      CNI_VERSION="v1.0.1"
+      RELEASE_VERSION="v0.4.0"
+      DOWNLOAD_DIR=/opt/bin
+      RELEASE="$(curl -sSL https://dl.k8s.io/release/stable.txt)"
+
+      mkdir -p $DOWNLOAD_DIR
+      mkdir -p /opt/cni/bin
+      mkdir -p /etc/systemd/system/kubelet.service.d
+
+      # Install tools
+      curl -sSL -o $DOWNLOAD_DIR/jq "https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64"
+      chmod +x $DOWNLOAD_DIR/jq
+      curl -sSL -o $DOWNLOAD_DIR/flatcar_developer_container.bin.bz2 "https://stable.release.flatcar-linux.net/amd64-usr/current/flatcar_developer_container.bin.bz2"
+      bzcat $DOWNLOAD_DIR/flatcar_developer_container.bin.bz2 > $DOWNLOAD_DIR/flatcar_developer_container.bin
+
+      curl -sSL "https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}/cni-plugins-linux-amd64-${CNI_VERSION}.tgz" | tar -C /opt/cni/bin -xz
+      curl -sSL "https://raw.githubusercontent.com/kubernetes/release/${RELEASE_VERSION}/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service" | sed "s:/usr/bin:${DOWNLOAD_DIR}:g" | tee /etc/systemd/system/kubelet.service
+      curl -sSL "https://raw.githubusercontent.com/kubernetes/release/${RELEASE_VERSION}/cmd/kubepkg/templates/latest/deb/kubeadm/10-kubeadm.conf" | sed "s:/usr/bin:${DOWNLOAD_DIR}:g" | tee /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+      curl -sSL --remote-name-all https://storage.googleapis.com/kubernetes-release/release/${RELEASE}/bin/linux/amd64/{kubeadm,kubelet,kubectl}
+
+      chmod +x {kubeadm,kubelet,kubectl}
+      mv {kubeadm,kubelet,kubectl} $DOWNLOAD_DIR/
+
+      # Disable kernel print rate limiting for syslog messaging
+      sysctl -w kernel.printk_ratelimit=0
+      sysctl -w kernel.printk_ratelimit_burst=0
+
+      # Import the test image for security profiles operator 
+      docker load -i /vagrant/image.tar
+
+      # Install cri-o
+      # /vagrant/hack/ci/install-cri-o.sh
```

Comment:
If this is not used, could you remove this? It's confusing for readers

Reply:
I removed these commented lines.